package.json

```diff
@@ -9,7 +9,6 @@
     "build-apps": "lerna run build --scope *-sample*",
     "lint-packages": "lerna run lint --ignore *-sample*",
     "lint-apps": "lerna run lint --ignore @sitecore-jss/*",
-    "test-packages": "lerna run test --ignore *-sample*",
     "test": "lerna run test",
```

Comment:
When running `test` locally, Lerna says it's attempting to run `test` in 18 packages but there are only 17, and the process does not exit after the 17th package's tests are complete.

Is it still needed, or can we remove this script?

Reply:
Removed, as we decided